LICENSE.AGPL

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
This was exactly what I was trying to say: With the current code/architecture having different validations for (BFT-)CoSi without modifying the code isn't possible. Thanks for the clarification.